### ISSUE TITLE
Fails on windows: error unknown switch 'd' On R studio Version 1.1.463 windows 10, Git-2.21.0-64-bi

### ISSUE THREAD

**Issue by Emma19901** [conventional-changelog/standard-version#360]
On R studio Version 1.1.463 windows 10. I am trying to sync git with R studio. After downloading Git-2.21.0-64-bit and following every step when I try to create a new project I receive this message. 

error: unknown switch `d'
usage: git clone [<options>] [--] <repo> [<dir>]

    -v, --verbose         be more verbose
    -q, --quiet           be more quiet
    --progress            force progress reporting
    -n, --no-checkout     don't create a checkout
    --bare                create a bare repository
    --mirror              create a mirror repository (implies bare)
    -l, --local           to clone from a local repository
    --no-hardlinks        don't use local hardlinks, always copy
    -s, --shared          setup as shared repository
    --recurse-submodules[=<pathspec>]
                          initialize submodules in the clone
    -j, --jobs <n>        number of submodules cloned in parallel
    --template <template-directory>
                          directory from which templates will be used
    --reference <repo>    reference repository
    --reference-if-able <repo>
                          reference repository
    --dissociate          use --reference only while cloning
    -o, --origin <name>   use <name> instead of 'origin' to track upstream
    -b, --branch <branch>
                          checkout <branch> instead of the remote's HEAD
    -u, --upload-pack <path>
                          path to git-upload-pack on the remote
    --depth <depth>       create a shallow clone of that depth
    --shallow-since <time>
                          create a shallow clone since a specific time
    --shallow-exclude <revision>
                          deepen history of shallow clone, excluding rev
    --single-branch       clone only one branch, HEAD or --branch
    --no-tags             don't clone any tags, and make later fetches not to follow them
    --shallow-submodules  any cloned submodules will be shallow
    --separate-git-dir <gitdir>
                          separate git dir from working tree
    -c, --config <key=value>
                          set config inside the new repository
    -4, --ipv4            use IPv4 addresses only
    -6, --ipv6            use IPv6 addresses only
    --filter <args>       object filtering

**Comment by jbottigliero** [comment]
Hey @Emma19901 – you are seeing this error when you are attempting to run `standard-version` in the environment combination you mentioned? The output you provided looks like the stdout of `git clone -h`.

**Comment by shreyas-afk** [comment]
TLDR: use .git at the end of your repo link

yes i got the same error when i tried to link my R project to git. Here's how i solved it
1. open GitHub and copy the link to your repo
2. paste the link when Rstudio asks for repo link and append .git at the end of it

Initailly i had pasted the link open in my browser -> https://github.com/shreyas-afk/reponame onto Rstudio which put out this error, Then i used this -> https://github.com/shreyas-afk/reponame.git which solved it
